package.json

```diff
@@ -63,11 +61,9 @@
     "pify": "^4.0.0",
     "request": "^2.87.0",
     "retry-request": "^4.0.0",
-    "stream-events": "^1.0.4",
     "through2": "^2.0.3"
   },
   "devDependencies": {
```

Comment:
IMHO this should be included so we can run `repo-tools generate all` every once in a while or we'll end up with the wrong list of contributors etc.